src/zones/dbm.rs

```diff
@@ -1296,6 +1369,9 @@ impl<T: DBMState> Display for DBM<T> {
 
 #[cfg(test)]

```

Comment:
make a comment showing how they work. example for dbm_shrink_expand
// 0, 1, 2, 3, 4
// 5, 6, 7, 8, 9
// 10, 11, 12, 13, 14
// 15, 16, 17, 18, 19
//, 20, 21, 22, 23, 24
is redrawn into
// 0, 1, 4, inf, inf
// 5, 0, 9, inf, inf
// 10, 11, 0, inf, inf
// 0, 0, 0, 0, inf
//  0, 0, 0, 0, 0